packages/playwright-ct-core/src/tsxTransform.ts

```diff
@@ -141,6 +143,24 @@ function collectJsxComponentUsages(node: T.Node): Set<string> {
   return names;
 }
 
+function collectClassMountUsages(node: T.Node): Set<string> {
```

Comment:
There's an open issue regarding this: https://github.com/microsoft/playwright/issues/29544. It would be great if this could be submitted directly to the main branch of Playwright as a separate PR. Other frameworks also suffer from this ATM. Let me know if I need to take care of that

Reply:
fixed in https://github.com/microsoft/playwright/pull/30269